assets/src/blocks/TopicLink/TopicLinkBlock.js

```diff
@@ -0,0 +1,51 @@
+import {TopicLinkEditor} from './TopicLinkEditor.js';
+
+const {registerBlockType} = wp.blocks;
+
+const BLOCK_NAME = 'planet4-blocks/topic-link';
+
+export const registerTopicLinkBlock = () => {
+  registerBlockType(BLOCK_NAME, {
+    title: 'Topic Link',
+    icon: 'migrate',
+    category: 'planet4-blocks',
+    supports: {
+      html: false, // Disable "Edit as HTMl" block option.
+    },
+    // This attributes definition mimics the one in the PHP side.
+    attributes: {
+      categoryId: {
+        type: 'number',
+        default: '',
+      },
+      categoryLink: {
+        type: 'string',
+        default: '',
+      },
+      categoryName: {
+        type: 'string',
+        default: '',
+      },
+      imageId: {
+        type: 'number',
+        default: '',
```

Comment:
Do we really need to set `''` by default if it is a `number`?

Reply:
Fixed!